### ISSUE TITLE
Empty welcome window

### ISSUE THREAD

**Issue by vkhodygo** [CellProfiler/CellProfiler#4431]
**Describe the bug**
When I start CellProfiler I get a welcome window which is I assume should contain some useful information.
However, it's blank empty. 

**To Reproduce**
1. Install `CellProfiler==4.2.0`
2. Run it 

**Expected behavior**
It's either should have some text or can be safely removed.

**Screenshots**
![Screenshot from 2021-07-15 14-07-37](https://user-images.githubusercontent.com/6596606/125794987-7b94cbaa-6923-48e0-b128-b6185b062a21.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04 VM
 - CellProfiler Version 4.2.0
 - source
 - pip install

**Comment by bethac07** [comment]
Yup, this is a subset of https://github.com/CellProfiler/CellProfiler/issues/4415 ; we've been hampered by the fact that we don't have an Ubuntu 20.04 GUI environment for testing on. I've been working on setting up a VM for that and will hopefully be able to dig into it further soon.

**Comment by vkhodygo** [comment]
@bethac07 I do have a VM with Ubuntu 20.04 onboard and I can run some tests for you.
